contributing_to_docs/doc_guidelines.adoc

```diff
@@ -367,7 +367,9 @@ An `include` directive must always be on its own line.
 ====
 \include::modules/module-filename.adoc[leveloffset=+1] <1>
 ====
+----
```

Comment:
I don't think this is the right fix. I think the `====` around the example above need to be changed to `----`. and the leading `\` removed. Then I think it will render properly as a callout